docs_user/modules/con_changes-to-cephFS-via-NFS.adoc

```diff
@@ -2,21 +2,14 @@
 
 = Changes to CephFS through NFS
 
```

Comment:
I want to suggest moving this module to the "Storage requirements" module in "Chapter 1 : Planning the new deployment."  I'm trying to keep "planning" type of information in Chapter 1.